src/plugins/video/info.js

```diff
@@ -37,6 +41,6 @@ export default class Video {
   }
 
   getCdnResources(content) {
-    return content.type === SOURCE_TYPE.internal && content.url ? [content.url] : [];
+    return content.sourceType === SOURCE_TYPE.internal && content.sourceUrl ? [content.sourceUrl] : [];
```

Comment:
The poster image - if set - is also a CDN resource but is missing here.

Reply:
Good catch, thanks! Updated